css/style.css

```diff
@@ -661,6 +1425,270 @@ body {
+    width: 260px;
+    margin-bottom: 18px;
+  }
+
+  .video__text {
+    font-size: 14px;
+    line-height: 17px;
+  }
+
+  .video__frame {
+    width: 293px;
+    height: 230px;
+    background: url(../img/video-bg-720.png) no-repeat;
+    margin-top: 1px;
+  }
+
+  .video__btn {
+    height: 19px;
+    width: 19px;
+    margin-bottom: 71px;
+    margin-right: 10px;
+  }
+
+  .video__btn::before {
+    left: 8px;
+    bottom: 7px;
+    background: url(../icons/Rectangle-375.png);
+    width: 5px;
+    height: 5px;
+  }
+
+  .stories {
+    background: linear-gradient(2.6deg, #fff 19.5%, #000 20%);
+    margin-bottom: 85px;
+  }
+
+  .stories__header {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 28px;
+  }
+
+  .stories__card-wrapper {
+    max-width: 374px;
+    width: 374px;
+    padding-left: 0;
+    margin-bottom: 103px;
+  }
+
+  .stories-card:first-child {
+    padding-right: 0;
+    padding-left: 49px;
+  }
+
+  .stories-card:nth-child(2) {
+    display: none;
+  }
+
+  .stories-card__img_768 {
+    height: 148px;
+  }
+
+  .stories-card__header {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .stories-card__text {
+    font-size: 14px;
+    line-height: 17px;
+    width: 282px;
+    margin-bottom: 25px;
+  }
+
+  .stories-card__link {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .stories-card__footer {
+    width: 279px;
+  }
+
+  .stories-card__link::after {
+    left: 97px;
+    bottom: 0;
+    background: url(../icons/arrow-375.png);
+    width: 39px;
+    height: 10px;
+  }
+
+  .stories__subheader {
+    font-size: 25px;
+    line-height: 24px;
+    padding-right: 52px;
+    margin-bottom: 11px;
+  }
+
+  .stories__gallery_1024 {
+    display: none;
+  }
+
+  .stories__gallery_375 {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 0 47px 0 47px;
+  }
+
+  .gallery__item_375:hover {
+    opacity: .4;
+    cursor: pointer;
+  }
+
+  .footer {
+    min-height: 897px;
+  }
+
+  .footer__box-wrapper {
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    padding: 0;
+    padding-top: 38px;
+  }
+
+  .footer__box:first-child,
+  .footer__box:nth-child(2),
+  .footer__box:nth-child(3) {
+    margin: 0;
+  }
+
+  .footer__box:first-child {
+    order: 1;
+    margin-bottom: 42px;
+  }
+
+  .footer__box:nth-child(2) {
+    order: 3;
+    margin-bottom: 41px;
+  }
+
+  .footer__box:nth-child(3) .footer__subheader {
+    margin-bottom: 7px;
+  }
+
+  .footer__box:nth-child(2) .footer__link {
+    line-height: 15px;
+  }
+
+  .footer__box:nth-child(3) {
+    order: 2;
+    margin-bottom: 44px;
+  }
+
+  .footer__box:nth-child(3) .footer__link {
+    line-height: 14px;
+  }
+
+  .footer__box:last-child {
+    order: 4;
+  }
+
+  .footer__box:last-child .footer__subheader {
+    margin-bottom: 10px;
+  }
+
+  .footer__subheader {
+    font-size: 15px;
+    line-height: 14px;
+    margin-bottom: 4px;
+  }
+
+  .footer__link {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".footer__link" to come before selector ".footer__box:last-child .footer__link" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5612618)